share/revng/test/tests/analysis/ImportFromCAnalysis/rft-with-a-stack-return-value/expected-error.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Cannot attach _STACK to return values in a RawFunctionDefinition.
+import-from-c: Return value should have already been parsed.
```

Comment:
Again on the wording of the error message. What about `Missing definition of return type`?

Reply:
I went with `Unable to parse the type of the return value.`

Reply:
:+1: